src/murmur/ServerUser.h

```diff
@@ -139,6 +141,8 @@ class ServerUser : public Connection, public User {
 		QList<int> qlCodecs;
 		bool bOpus;
 
+		QList<VoiceProtocol> qlSupportedVoiceProtocols;
```

Comment:
This field is added with the aim of providing an interface so other APIs or the UserState window can query the supported cipher/protocol list of a client.